### PR TITLE
DDF-1880 Added STANAG 4559 Federated Source

### PR DESCRIPTION
**Pull request by bdeining**
@kcwire 
@jrnorth 
@dcruver 
@jaymcnallie 
@shane-voisard 
@jlcsmith

### CODE REVIEW

**Comment by bdeining** [comment]
This PR is built on top of DDF-1864.  After discussion, STANAG 4559 stuff is going to be moved to a different project. This PR won't be merged, but will be left open for code review until the new project is created.

File: catalog/nato/stanag4559/catalog-stanag4559-common/src/main/java/ddf/catalog/nato/stanag4559/common/Stanag4559Constants.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            "SYSTEM DEPLOYMENT STATUS",
+            "SYSTEM SPECIFICATIONS",
+            "TACTICAL SYMBOL",
+            "TASK",
+            "TDL DATA",
+            "VIDEO");
+
+    public static final Set<ContentType> CONTENT_TYPES = setContentStrings();
+
+    public static Set<ContentType> setContentStrings() {
```
Comment (jlcsmith) [inline catalog/nato/stanag4559/catalog-stanag4559-common/src/main/java/ddf/catalog/nato/stanag4559/common/Stanag4559Constants.java]: I believe you can use Arrays or CollectionUtils to do this logic.

File: catalog/nato/stanag4559/catalog-stanag4559-source/pom.xml [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <!-- Override Checkstyle for the MockStanag4559 Server -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <version>2.17</version>
```
Comment (jlcsmith) [inline catalog/nato/stanag4559/catalog-stanag4559-source/pom.xml]: don't think you need the version here as it should come from dependencyManagement

File: catalog/nato/stanag4559/catalog-stanag4559-source/pom.xml [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Import-Package>
```
Comment (jlcsmith) [inline catalog/nato/stanag4559/catalog-stanag4559-source/pom.xml]: You should be able to remove Import-Package in this case

File: catalog/nato/stanag4559/catalog-stanag4559-source/src/main/java/ddf/catalog/nato/stanag4559/source/Stanag4559FilterFactory.java [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import ddf.catalog.nato.stanag4559.common.GIAS.AttributeInformation;
+import ddf.catalog.nato.stanag4559.common.GIAS.AttributeType;
+
+public class Stanag4559FilterFactory {
+
+    public static final String METADATA_CONTENT_TYPE = "metadata-content-type";
```
Comment (jlcsmith) [inline catalog/nato/stanag4559/catalog-stanag4559-source/src/main/java/ddf/catalog/nato/stanag4559/source/Stanag4559FilterFactory.java]: use Metacard.CONTENT_TYPE instead

**Comment by bdeining** [comment]
https://github.com/codice/alliance/pull/1

**Pull request closed without merging**